dist/images/ovnkube.sh

```diff
@@ -1030,6 +1034,258 @@ ovn-master() {
+    ${ovnkube_metrics_tls_opts} \
+    ${multicast_enabled_flag} \
+    ${ovn_acl_logging_rate_limit_flag} \
+    ${egressip_enabled_flag} \
+    ${egressip_healthcheck_port_flag} \
+    ${egressfirewall_enabled_flag} \
+    ${egressqos_enabled_flag} \
+    ${ovnkube_config_duration_enable_flag} \
+    ${multi_network_enabled_flag} \
+    --metrics-bind-address ${ovnkube_master_metrics_bind_address} \
+    --host-network-namespace ${ovn_host_network_namespace} &
+
+  echo "=============== ovn-network-controller-manager ========== running"
+  wait_for_event attempts=3 process_ready ovnkube-network-controller-manager
+
+  process_healthy ovnkube-network-controller-manager
+  exit 9
+}
+
+# run ovnkube --cluster-manager.
+ovn-cluster-manager() {
+  trap 'kill $(jobs -p); exit 0' TERM
+  check_ovn_daemonset_version "3"
+
+  echo "=============== ovn-cluster-manager (wait for ready_to_start_node) ========== MASTER ONLY"
+  wait_for_event ready_to_start_node
+
+  hybrid_overlay_flags=
+  if [[ ${ovn_hybrid_overlay_enable} == "true" ]]; then
+    hybrid_overlay_flags="--enable-hybrid-overlay"
+    if [[ -n "${ovn_hybrid_overlay_net_cidr}" ]]; then
+      hybrid_overlay_flags="${hybrid_overlay_flags} --hybrid-overlay-cluster-subnets=${ovn_hybrid_overlay_net_cidr}"
+    fi
+  fi
+  echo "hybrid_overlay_flags: ${hybrid_overlay_flags}"
+
+  ovn_v4_join_subnet_opt=
+  if [[ -n ${ovn_v4_join_subnet} ]]; then
+      ovn_v4_join_subnet_opt="--gateway-v4-join-subnet=${ovn_v4_join_subnet}"
+  fi
+  echo "ovn_v4_join_subnet_opt: ${ovn_v4_join_subnet_opt}"
+
+  ovn_v6_join_subnet_opt=
+  if [[ -n ${ovn_v6_join_subnet} ]]; then
+      ovn_v6_join_subnet_opt="--gateway-v6-join-subnet=${ovn_v6_join_subnet}"
+  fi
+  echo "ovn_v6_join_subnet_opt: ${ovn_v6_join_subnet_opt}"
+
+  multicast_enabled_flag=
+  if [[ ${ovn_multicast_enable} == "true" ]]; then
+      multicast_enabled_flag="--enable-multicast"
+  fi
+  echo "multicast_enabled_flag: ${multicast_enabled_flag}"
+
+  multi_network_enabled_flag=
+  if [[ ${ovn_multi_network_enable} == "true" ]]; then
+	  multi_network_enabled_flag="--enable-multi-network"
+  fi
+  echo "multi_network_enabled_flag: ${multi_network_enabled_flag}"
+
+  ovnkube_cluster_manager_metrics_bind_address="${metrics_endpoint_ip}:9411"
+  echo "ovnkube_cluster_manager_metrics_bind_address: ${ovnkube_cluster_manager_metrics_bind_address}"
+
+  local ovnkube_metrics_tls_opts=""
+  if [[ ${OVNKUBE_METRICS_PK} != "" && ${OVNKUBE_METRICS_CERT} != "" ]]; then
+    ovnkube_metrics_tls_opts="
+        --node-server-privkey ${OVNKUBE_METRICS_PK}
+        --node-server-cert ${OVNKUBE_METRICS_CERT}
+      "
+  fi
+  echo "ovnkube_metrics_tls_opts: ${ovnkube_metrics_tls_opts}"
+
+  ovnkube_config_duration_enable_flag=
+  if [[ ${ovnkube_config_duration_enable} == "true" ]]; then
+    ovnkube_config_duration_enable_flag="--metrics-enable-config-duration"
+  fi
+  echo "ovnkube_config_duration_enable_flag: ${ovnkube_config_duration_enable_flag}"
```

Comment:
remove - this wont do anything in cluster-manager mode.

Reply:
Ack